xml/System.Diagnostics.Tracing/PollingCounter.xml

```diff
@@ -43,12 +43,15 @@
       <Docs>
         <param name="name">The name for this <see cref="T:System.Diagnostics.Tracing.PollingCounter" /> instance.</param>
         <param name="eventSource">The event source.</param>
-        <param name="metricProvider">To be added.</param>
+        <param name="metricProvider">A method that will be used to poll the value.</param>
         <summary>Initializes a new instance of the <see cref="T:System.Diagnostics.Tracing.PollingCounter" /> class.</summary>
         <remarks>
           <format type="text/markdown"><![CDATA[  
+<xref:System.Diagnostics.Tracing.PollingCounter> is a variant of <xref:System.Diagnostics.Tracing.EventCounter> where instead of the writer having to explicitly call `WriteMetric()` each time a value wants to be tracked, it takes in a callback method to poll for values. It polls and reports a value retrieved from calling `metricProvider` exactly once per the interval requested by the listener. 
```

Comment:
```suggestion
## Remarks

<xref:System.Diagnostics.Tracing.PollingCounter> is a variant of <xref:System.Diagnostics.Tracing.EventCounter> where instead of the writer having to explicitly call `WriteMetric()` each time a value wants to be tracked, it takes in a callback method to poll for values. It polls and reports a value retrieved from calling `metricProvider` exactly once per the interval requested by the listener. 
```